test/zdtm/static/sigaltstack.c

```diff
@@ -1,6 +1,5 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include <signal.h>
```

Comment:
why do we not need to include signal.h? sigaltstack, sigaction need it, don't they?

Reply:
This was removed by accident. I added it back.